### PR TITLE
Add pkg-config and curl Build-Depends

### PR DESCRIPTION
**Pull request by marksvc**
* Since pkg-config is needed, and wget has trouble with a certificate.

### CODE REVIEW

File: debian/control [hunk 1]
```diff
@@ -7,6 +7,8 @@ Build-Depends: debhelper (>= 8.0.0),
  libmono-winforms2.0-cil,
  libmono-cairo2.0-cil,
  geckofx29, unzip, wget,
+ pkg-config,
+ curl,
```
Comment (jasonleenaylor) [inline debian/control]: why does it need wget and curl?

**Comment by neilmayhew** [comment]
:+1:

File: debian/control (hunk 1 above)
Comment (marksvc) [inline debian/control]: It downloads various dependencies.

Comment (marksvc) [inline debian/control]: Oh, it might not need wget AND curl. It didn't occur to me to remove 'wget', but I could if that's thought to be a good idea.

Comment (marksvc) [inline debian/control]: I am trying a package build without the 'wget' Build-Depends.

Comment (marksvc) [inline debian/control]: The package build fails without 'wget', so let's include both curl and wget.

**Pull request merged**